### PR TITLE
Doc 4161 - Merge/delete Corda Enterprise Edition 4.4 release notes

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
There's two sets of release notes for Enterprise Edition 4.4: what looks like a standard set here:[Corda Enterprise Edition 4.4 release notes](https://docs.r3.com/en/platform/corda/4.4/enterprise/release-notes-enterprise.html)and another set here:[Corda release notes](https://docs.r3.com/en/platform/corda/4.4/enterprise/release-notes.html)They look like they were both added by a technical writer last year in a copy.
The second link has information that was included in the Open Source release notes. So either (1) it was added to the Enterprise section as that information also needs to be in the Enterprise Edition release notes but rather than merge the notes they simply kept two files, or (2) it was just added by mistake and the Enterprise Edition release notes in the first link are OK as-is.

I've taken what I think needs to be included from release-notes.md and added it to release-notes-enterprise.md, then deleted release-notes.md.

### CODE REVIEW

File: content/en/platform/corda/4.4/enterprise/node/operating/cm-upgrading-node.md [hunk 1]
```diff
@@ -296,7 +296,7 @@ The option `-b` points to the base directory (with a `node.conf` file, and *driv
 ## Step 4. Replace `corda.jar` with the new version
 
 Replace the `corda.jar` with the latest version of Corda.
-Make sure it’s available on your path, and that you’ve read the [release notes](../../release-notes.md). Pay particular attention to which version of Java this node requires.
+Make sure it’s available on your path, and that you’ve read the [release notes](../../release-notes-enterprise.md). Pay particular attention to which version of Java this node requires.
```
Comment (yarivus) [inline content/en/platform/corda/4.4/enterprise/node/operating/cm-upgrading-node.md:299]: should this change to .html ?

File: content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md [hunk 2]
```diff
@@ -19,6 +19,11 @@ weight: 10
 
 # Corda Enterprise Edition 4.4 release notes
 
+Corda 4.4 lays the foundation of a new open-core approach for the Corda codebase. This involved a refactoring of the main functional components of Corda. Please consult cordapp-overview.rst to get an overview of the practical impact on CorDapp development.
```
Comment (yarivus) [inline content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md:22]: ' open-core ' ?

File: content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md [hunk 3]
```diff
@@ -232,66 +237,48 @@ Corda Enterprise Edition 4.4 supports Linux for production deployments, with Win
...
 * The notary can now be registered using its X500 name, as an alternative to providing a node info file. This allows the notary to be added to the network parameters before the notary is registered, and avoids the need to copy the node info file around between notary workers
 * HA notary workers can retrieve the notary’s service certificate from the network map service, avoiding the need to manually copy it around between the various workers
 * HA notary workers check they have access to the shared notary service key and certificate before they register with the notary
-
-
-
-#### Corda Health Survey improvements
-
+#
+### Corda Health Survey improvements
```
Comment (yarivus) [inline content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md:278]: is this change intended ?

File: content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md [hunk 4]
```diff
@@ -302,33 +289,124 @@ Furthermore, we have improved the overall usability of the tool by adding suppor
...
 
 ### Upgrade notes
 
 From Corda Enterprise Edition 4.4 onwards, we are moving towards an open core strategy. Common APIs shared by Corda Enterprise will only be available in Corda Open Source. Therefore, any CorDapps written against Corda Enterprise Edition 4.4 or later will have to depend on the open source version of `corda-core`.
 
-As per previous major releases, we have provided a [comprehensive upgrade guide](../../../../../en/platform/corda/4.4/enterprise/app-upgrade-notes-enterprise.md) to ease the upgrade of CorDapps to Corda Enterprise Edition 4.4.
+As per previous major releases, we have provided a [comprehensive upgrade guide](../../../../../en/platform/corda/4.4/enterprise/app-upgrade-notes-enterprise.md) to ease the upgrade of CorDapps to Corda Enterprise Edition 4.4.
```
Comment (yarivus) [inline content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md:412]: should be .html ?

**Review by yarivus — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.4/enterprise/node/operating/cm-upgrading-node.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.4/enterprise/node/operating/cm-upgrading-node.md:299]: Done.

File: content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md:22]: I presume it refers to this:

https://en.wikipedia.org/wiki/Open-core_model

Shall I say "new open-core model approach"?

File: content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md (hunk 3 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md:278]: Thanks, fixed that.

File: content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md (hunk 4 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md:412]: Changed to HTML.

File: content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md (hunk 2 above)
Comment (yarivus) [reply content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md:22]: I think it's better ;)

**Review by yarivus — APPROVED** [review]
LGTM

**Review by AnnaK-R3 — APPROVED** [review]
